c7n/resources/sagemaker.py

```diff
@@ -216,7 +216,7 @@ class RemoveTagNotebookInstance(RemoveTag):
 
             policies:
               - name: sagemaker-notebook-remove-tag
-                resource: notebook-instabce
+                resource: notebook-instance
```

Comment:
thanks for fixing the typo, but the actual resource is `sagemaker-notebook`

Reply:
ugh, find/replace didn't catch that one because it was misspelled...